src/System.Management.Automation/logging/LogProvider.cs

```diff
@@ -102,6 +102,13 @@ internal LogProvider()
         /// <param name="previousValue"></param>
         internal abstract void LogSettingsEvent(LogContext logContext, string variableName, string value, string previousValue);
 
+        /// <summary>
+        /// Provider interface function for logging AmsiUtil State event.
+        /// </summary>
+        /// <param name="State"></param>
+        /// <param name="context"></param>
+        internal abstract void LogAmsiUtilStateEvent(string State, string context);
```

Comment:
```suggestion
        /// <param name="state"></param>
        /// <param name="context"></param>
        internal abstract void LogAmsiUtilStateEvent(string tate, string context);
```

please also describe the parameters in the comments